src/index.ts

```diff
@@ -23,6 +23,7 @@ const cjkPunctuations = unicode({
     'Kanbun',
     'CJK_Strokes',
     'Enclosed_CJK_Letters_And_Months',
+    'Katakana',
```

Comment:
Hello @tats-u, I just got access to this repo, I'll take care of this PR.

There seems more than just U+30FB and U+30A0, are you sure there should all treat as punctuations?

https://en.wikipedia.org/wiki/Katakana_(Unicode_block)

Reply:
As you may suspect, many of them are just letters. I have to fix.

Reply:
@fisker I remembered I've thought all of the non-punctuation characters in the Katakana block are included in `cjkLetters`.

Reply:
All right, I'll take a look tomorrow.

Reply:
![image](https://github.com/user-attachments/assets/43b4456b-f802-41bd-aa15-9ac7041e9c18)